ceph/ceph_admin/bootstrap.py

```diff
@@ -419,6 +419,19 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
             self.shell(
                 args=["ceph", "config", "set", "global cluster_network", cluster_nws]
             )
+        if rhbuild.split(".")[0] >= "8":
```

Comment:
Thanks @pdhiran for addressing this issue.

IMHO, we could use `LooseVersion` instead of `split`. Both the approaches work though the former is easier to read.

```
self.cluster.rhcs_version >= LooseVersion("8.0")
```

Reply:
Thanks for the suggestion. Made the change.